server/handler/sessionHelper.go

```
@@ -70,7 +71,10 @@ func (s *SessionHelper) SetupSession(r *http.Request, w http.ResponseWriter) err
 		return err
 	}
 	aSession.Options.HttpOnly = true
-	aSession.Save(r, w)
+	err = aSession.Save(r, w)
```

We need to return this error.

---

For sure.